privacyidea/lib/policy.py

```diff
@@ -2117,6 +2118,14 @@ def get_static_policy_definitions(scope=None):
                 'type': 'int',
                 'desc': _('Limit the number of allowed tokens in a realm.'),
                 'group': GROUP.TOKEN},
+            ACTION.REQUIRE_DESCRIPTION: {
+                'type': 'str',
+                'desc': _('During the rollout process, this policy makes the '
+                          'description required for all selected token-types.'),
```

Comment:
I think we usually write tokentype in one word:

"token-types" -> "tokentypes".

Reply:
fixed in 38630095c78fff51b7fda94e221140375e04ac48.